File: Java_Projects/JavaProgramming/Polymorphism/Example2.java
package JavaProgramming.Polymorphism;

/*
 * Method Overloading: changing no. of arguments
 * In this example, we have created two methods, first add() method performs
 * addition of two numbers and second add method performs addition of three
 * numbers.
 * 
 * In this example, we are creating static methods
 * so that we don't need to create instance for calling methods.
 */
class Adder {
    static int add(int a, int b) {
        return a + b;
    }

    static int add(int a, int b, int c) {
        return a + b + c;
    }
}

class TestOverloading1 {
    public static void main(String[] args) {
        System.out.println(Adder.add(11, 11));
        System.out.println(Adder.add(11, 11, 11));
    }
}

File: C#_Projects/C# Programming/Polymorphism/Example2.cs
// C# Member Overloading Example: By changing data type of arguments

using System;  
public class Cal{  
    public static int add(int a, int b){  
        return a + b;  
    }  
    public static float add(float a, float b)  
    {  
        return a + b;  
    }  
}  
public class TestMemberOverloading  
{  
    public static void Main()  
    {  
        Console.WriteLine(Cal.add(12, 23));  
        Console.WriteLine(Cal.add(12.4f,21.3f));  
    }  
}  